src/oscar/apps/basket/middleware.py

```diff
@@ -7,7 +7,7 @@
 from oscar.core.compat import MiddlewareMixin, user_is_authenticated
 from oscar.core.loading import get_class, get_model
 
-Applicator = get_class('offer.utils', 'Applicator')
+Applicator = get_class('offer.applicator', 'Applicator')
```

Comment:
In fact `offer.utils` already backward compatible, since it has import of `Applicator` class - https://github.com/django-oscar/django-oscar/blob/master/src/oscar/apps/offer/utils.py#L6.  Will add a note to the release notes about proper location too.